deepchem/utils/differentiation_utils/misc.py

```diff
@@ -131,3 +131,43 @@ def assert_runtime(cond, msg=""):
     """
     if not cond:
         raise RuntimeError(msg)
+
+
+# repr helper functions
+def indent(s, nspace):
+    """give indentation of the second line and next lines
```

Comment:
Please add a note this function is used to generate string representations of LinearOperator (and any other classes it is used for)